docs/TASK.md

```diff
@@ -307,7 +285,7 @@ export interface TrainingInformation {
   // batchSize: batch size of training data
```

Comment:
we are using `typedoc` in a few places, is the doc pushed somewhere? can we instead reference the documented `TrainingInformation` to avoid out-of-sync doc & code?

Reply:
I have no idea about typedoc I didn't read anything about it so far. And yes good idea I'll point to TrainingInformation.